examples/tcp-socket-listener-client/tcp_socket_client.bal

```diff
@@ -24,16 +24,28 @@ service ClientService = service {
     }
 
     // This is invoked when the server sends any content.
-    resource function onReadReady(socket:Caller caller, byte[] content) {
-        io:ReadableByteChannel byteChannel = io:createReadableChannel(content);
-        io:ReadableCharacterChannel characterChannel =
-                        new io:ReadableCharacterChannel(byteChannel, "UTF-8");
-        var str = characterChannel.read(25);
-        if (str is string) {
-            io:println(untaint str);
+    resource function onReadReady(socket:Caller caller) {
+        var result = caller->read();
+        if (result is (byte[], int)) {
+            var (content, length) = result;
+            if (length > 0) {
+                io:ReadableByteChannel byteChannel =
+                    io:createReadableChannel(content);
+                io:ReadableCharacterChannel characterChannel =
+                    new io:ReadableCharacterChannel(byteChannel, "UTF-8");
+                var str = characterChannel.read(25);
+                if (str is string) {
```

Comment:
Do we need this if-else statement if we are just simply printing the value in both cases?